github/Branch.py

```diff
@@ -71,6 +71,13 @@ def protected(self):
         """
         return self._protected.value
 
+    @property
```

Comment:
I mean, I get why you're adding it, but we also don't check it anywhere at all ... Not even tests.

Reply:
As an attribute returned by GitHub, I thought it made sense to expose. I'm not sure if you are asking for this to be removed or for tests to check it.